src/FANET/FanetStation.hpp

```diff
@@ -0,0 +1,99 @@
+/*
+Copyright_License {
+
+  XCSoar Glide Computer - http://www.xcsoar.org/
+  Copyright (C) 2000-2016 The XCSoar Project
+  A detailed list of copyright holders can be found in the file "AUTHORS".
+
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU General Public License
+  as published by the Free Software Foundation; either version 2
+  of the License, or (at your option) any later version.
+
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with this program; if not, write to the Free Software
+  Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+}
+*/
+
+#ifndef XCSOAR_FANET_STATIONS_HPP
+#define XCSOAR_FANET_STATIONS_HPP
+
+#include "util/StaticString.hxx"
+#include <type_traits>
+#include <tchar.h>
+#include "FanetAddress.hpp"
+#include "NMEA/Validity.hpp"
+#include "Geo/GeoPoint.hpp"
+
+#define FANETFRAMESERVICE_INET			        7
+#define FANETFRAMESERVICE_TEMP			        6
+#define FANETFRAMESERVICE_WIND			        5
+#define FANETFRAMESERVICE_HUMIDITY		      4
+#define FANETFRAMESERVICE_PRESSURE		      3
+#define FANETFRAMESERVICE_REMOTECFGSUPPORT	2
+#define FANETFRAMESERVICE_SOC			          1
+#define FANETFRAMESERVICE_EHEADER           0
+
+#define IS_NTH_BIT_SET(byte, bit) byte & (1<<bit)
+
+struct FanetStation {
+  uint8_t type;
```

Comment:
What does this field mean?

Reply:
Removed, decoding is now done in Parser.cpp so its not needed